### ISSUE TITLE
[Feature Request]  Automatic flush of expired temporary rules (after a period)

### ISSUE THREAD

**Issue by BetweenTheKeyboardAndChair** [evilsocket/opensnitch#824]
I'd like temporary rules, e.g. block/allow for 5 minutes, to be automatically deleted after a specified period.  

The behavior and UI could be similar to the automatic flushing of events after N days.

(I asked for this in another conversation quite a while back.  I don't think it ever got its own feature request.)

**Comment by gustavo-iniguez-goya** [comment]
Thank you @BetweenTheKeyboardAndChair , I looked for this request but I didn't find it, so I only added more options to discard temporary rules.

The new option could be: 
```
[x] Delete temporary rules: [ 5m or less ]
[x] After                           +[ 2 ]- days.
```

**Comment by BetweenTheKeyboardAndChair** [comment]
That looks good.  Thank you.

**Comment by mixmastamyk** [comment]
Yes, I have to close the GUI app and restart it every so often to flush the temporary rules.  Also would like them gone a lot quicker than mentioned above.  Perhaps 5 minutes after they expire.

**Comment by mixmastamyk** [comment]
This would nice to be able to get into 2026 LTS releases, or will have to suffer with old rules for years.
